src/include/miopen/conv/wrw_invoke_params.hpp

```diff
@@ -38,28 +38,33 @@ struct WrWInvokeParams : InvokeParams
     Data_t workSpace;
     std::size_t workSpaceSize;
     bool gfx90aFp16alt;
+    bool deterministic;
```

Comment:
Why you would like to pass this attrib into Invoker? It's not used there except for `std::cout << ...`.

Reply:
Since this is designed within ConvDescAttribute, I just following the same design with `gfx90aFp16alt`, so I passed this value to all the `InvokeParams`. Do you think this is actually needed or not?

Reply:
Please do not pass this into Invokers.

Reply:
This is obviously not needed because you use it only for logging.